index.js

```diff
@@ -1 +1,130 @@
+const popup = document.querySelector(".popup");
 
+const inputFormAddTask = document.querySelectorAll(".form__input-task");
+const textareaFormAddTask = document.querySelector(".form__input-note");
+const openPopupButton = document.querySelector(".add-task__button");
+const closePopupButton = document.querySelector(".popup__close");
```

Comment:
внутри попапа, кнопки удобно сокращаться до btn:
```suggestion
const closePopupBtn = popup.querySelector(".popup__close");
```